flows.yaml

```diff
@@ -1898,6 +1901,8 @@ integrations:
                 description: |
                     Send an Email using Gmail.
         models:
+            OptionalBackfillSetting:
+                backfillPeriod: number
```

Comment:
I would put the unit in the name to avoid any confusion or even having to look at the doc. for example `backfillPeriodMs`

Reply:
Yup, thanks